src/apps/recommender/InspirationRecommender.tsx

```diff
@@ -41,7 +41,10 @@ const InspirationRecommender: FC = () => {
       <ul className="recommender__grid">
         {recommendedMaterials &&
           recommendedMaterials.search.works.map((work) => (
-            <RecommendMaterial work={work as Work} />
+            <RecommendMaterial
+              appTag="inspiration-recommender"
```

Comment:
What is the purpose of `appTag`? It is the type of a `GuardedAppId` but it is not, as far as I can see, using the guardedApp functionality.

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/440/files#diff-a1cd3c7df4bbe6dec8c1f3bb698cb32021057973376864601e5bffb80e630371R59-R62

It ends up in the simple-material, which creates a guarded request

Reply:
renamed